File: VBDotNet/src/Lion.vb
Namespace ClassZoo
    Public Class Lion 
        Inherits Animal
        Sub New(InputName As String, InputID As Integer, InputWeight As Single, InputHeight As Single)
            MyBase.New(InputName, InputID, InputWeight, InputHeight)
        End Sub

        Public Overrides Sub PrintSpecies()
            Console.WriteLine("Animal Species: Lion")
        End Sub 
        Public Overrides Sub AnimalSound()
            Console.WriteLine(Me.GetName() + " roars")
        End Sub
    End Class
End Namespace

File: C_Sharp/src/Lion.cs
namespace ClassZoo;

public class Lion : Animal 
{
    public Lion(string inputName, int inputID, float inputWeight, float inputHeight) 
    : base(inputName, inputID, inputWeight, inputHeight)
    {
    }

    public override void PrintSpecies()
    {
        Console.WriteLine("Animal Species: Lion");
    }
    public override void AnimalSound()
    {
        Console.WriteLine($"{this.GetName() + " roars"}");
    }
}